src/AppInstallerRepositoryCore/Microsoft/SQLiteIndexSource.h

```diff
@@ -79,4 +81,19 @@ namespace AppInstaller::Repository::Microsoft
         // Removes a package version from the source.
         void RemovePackageVersion(const Manifest::Manifest& manifest, const std::filesystem::path& relativePath);
     };
+
+    namespace details
+    {
+        // The base for the package objects.

```

Comment:
nit: seems like wrong description

Reply:
No, it isn't wrong, but it probably made more sense before I refactored it away from the things that inherit it.